UnityEngine.AssetBundles/Editor/AssetBundleBrowser/InspectTab/AssetBundleInspectTab.cs

```diff
@@ -223,15 +270,84 @@ public List<string> BundleList
         public void SetBundleItem(InspectTreeItem selected)
         {
             if (selected == null)
+            {
                 m_SingleInspector.SetBundle(null);
+            }
             else
-                m_SingleInspector.SetBundle(selected.bundle, selected.bundlePath);
+            {
+                AssetBundle bundle = this.LoadBundle(selected.bundlePath);
+                m_SingleInspector.SetBundle(bundle, selected.bundlePath);
+            }
         }
 
         [System.Serializable]
         public class InspectTabData
         {
             public string m_BundlePath = string.Empty;
         }
+
+        /// <summary>
+        /// Loads the bundle at the specified path.
+        /// Unloads previously loaded bundles if neccessary.
+        /// </summary>
+        /// <returns>Returns loaded bundle, null if it could not be loaded.</returns>
+        /// <param name="path">Path of bundle to load</param>
+        private AssetBundle LoadBundle(string path)
+        {
+            if (string.IsNullOrEmpty(path))
+            {
+                return null;
+            }
+
+            string bundleName = Path.GetFileNameWithoutExtension(path);
+            AssetBundle bundle = this.GetLoadedBundleByName(bundleName);
+            if (null != bundle)
```

Comment:
The problem I was trying to solve was to allow the user to select a bundle variant in the inspector tab without experiencing errors with previously loaded variants. 

I originally tried to index loaded asset bundles using the full path, but then found I would have to search through the map to find the correct bundle to unload. Thus, I decided to index by name instead of path to keep things simple.

This does have the side effect of loading/unloading bundles each time you re-select a bundle with the same name. I understand this is not optimal and am open to suggestions.

I think I should change the managing data structure to include a bucket of variants and only unload/reload the bundle if we are dealing with variants.

I'll give this a try and submit it w/ the next commit if it checks out.